src/odemis/gui/comp/legend.py

```diff
@@ -415,13 +415,16 @@ def on_paint(self, _):
         if self._value_range is None:
             return
 
-        # shared function with the export method
-        self._tick_list, self._vtp_ratio = calculate_ticks(self._value_range, self.ClientSize, self._orientation, self._tick_spacing)
+        offset_y = 0
```

Comment:
It used to be 10, now you're setting it to 0 by default. I think that's wrong.

Reply:
It gets set now in the default argument for the offset